# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/service-workers/service-worker/update-no-cache-request-headers.https.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/service-workers/service-worker/update-no-cache-request-headers.https.html.ini
--- a/testing/web-platform/meta/service-workers/service-worker/update-no-cache-request-headers.https.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/service-workers/service-worker/update-no-cache-request-headers.https.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,8 +1,9 @@
 [update-no-cache-request-headers.https.html]
   [headers in no-cache mode]
     expected:
-      if (os == "linux") and debug and not fission and webrender and not swgl: ["FAIL", "PASS"]
-      if (os == "linux") and debug and fission: ["FAIL", "PASS"]
-      if (os == "win") and not debug and webrender: ["FAIL", "PASS"]
+      if not fission and (processor == "x86_64") and (os == "win") and not debug and webrender: ["FAIL", "PASS"]
+      if not fission and (processor == "x86") and (os == "linux"): ["FAIL", "PASS"]
+      if fission and (os == "linux") and debug: ["FAIL", "PASS"]
+      if fission and (os == "win"): ["FAIL", "PASS"]
       [PASS, FAIL]
 